matrix_sdk/src/client.rs

```diff
@@ -429,6 +430,22 @@ impl Client {
         session.as_ref().cloned().map(|s| s.user_id)
     }
 
+    /// Fetches the display name of the owner of the client.
```

Comment:
I know there's not much going on here but I think adding an example for this may still be worth it. It allows people to glance at docs and remind them how stuff is used in the rust-sdk.

Reply:
should the examples be no_run or not?

Reply:
Yeah, `no_run` otherwise we might be doing HTTP requests.